docs/content/about/_index.md

```diff
@@ -15,6 +15,14 @@ aliases:
   - /about/goals/
 ---
 
+## Our Vision
+
+NIST and the community will maintain a set of OSCAL models for organizations and service providers to facilitate seamless, continuous security assessment by exchanging information about system implementation and analysis without vendor lock-in. ​
```

Comment:
Suggestion:  NIST and the community will mature and maintain a set of ...

Reply:
Sending up a commit with these recommendations integrated.